src/screens/NationDetailsContainer/NationDetailsScreen/index.js

```diff
@@ -41,12 +66,11 @@ class NationDetailsScreen extends Component {
           <View style={styles.titleContainer}>
             <View style={styles.titleBarLarge}>
               <Text style={styles.largeTitle}>{nation.nationName}</Text>
-              {console.log('joined nation: ', nation.joined)}
             </View>
           </View>
 
           <ScrollView>
-            {status !== 'draft' && this._buildStatusPanel(i18n.t(`screens.nationDetails.statusDescription.${status}`))}
+            {statusDescription != '' && this._buildStatusPanel(statusDescription)}
```

Comment:
That is better to use strict comparison to prevent unexpected type coercion. Another option is to use lodash `isEmpty` function.